src/OrbitCaptureClient/ApiEventProcessor.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitCaptureClient/ApiEventProcessor.h"
+
+#include "OrbitBase/Logging.h"
+
+using orbit_client_protos::TimerInfo;
+using orbit_grpc_protos::ApiEvent;
+
+ApiEventProcessor::ApiEventProcessor(CaptureListener* listener) : capture_listener_(listener) {}
+
+static inline TimerInfo TimerInfoFromEncodedEvent(const orbit_api::EncodedEvent& encoded_event,
```

Comment:
What is `inline` for here?

Reply:
As a hint to the compiler to inline the function. Is static and inline mutually exclusive?

Reply:
No no, but we basically never use it just to hint the compiler to try to inline (I guess that for a `static` function the compiler will probably inline anyway?). But I'm totally fine with having it if that's the intent.